### PR TITLE
[15.0][IMP] helpdesk_mgmt: Change toggle_active to web_ribbon

### PR DESCRIPTION
**Pull request by victoralmau**
Change `toggle_active` to `web_ribbon`.

This is already changed in v16.

Please @pedrobaeza can you review it?

@Tecnativa TT43041

### CODE REVIEW

**Comment by dalonsod** [comment]
Is right removing `button_box` div?

File: helpdesk_mgmt/views/helpdesk_ticket_category_views.xml [hunk 1]
```diff
@@ -23,20 +23,12 @@
                 <header>
                 </header>
                 <sheet>
-                    <div class="oe_button_box" name="button_box">
```
Comment (pedrobaeza) [inline helpdesk_mgmt/views/helpdesk_ticket_category_views.xml:26]: Let this as empty container for inheritance.

**Review by pedrobaeza — APPROVED** [review]

**Review by dalonsod — APPROVED** [review]
Only code review

**Pull request merged**